ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.less

```diff
@@ -78,6 +78,13 @@
   }
 }
 
+@{customElementNamespace}filter.is-range {
```

Comment:
Can you verify this maintains the consistent spacing that @hayleynorton worked on in this PR: https://github.com/codice/ddf/pull/5118

Reply:
Yes, this mirrors what the between filter does on date ranges - but with an even smaller spacing (10x instead of 24x) of the query.